.gitignore

```diff
@@ -4,3 +4,5 @@ qupla.iws
 out
 .idea
 build
+.gradle
+.DS_Store
```

Comment:
This change is not related to the subject you managed to propose.

Reply:
Sure. But many developers use both a local gradle distribution, and Mac OS. Changing the .gitignore is fairly common, and having a proper .gitignore for all users is standard practice on public repositories. For another example, see the changes that were approved in PR #23, or read more about gitignore files at https://git-scm.com/docs/gitignore. There are also some standard .gitignore files listed [here](https://github.com/github/gitignore). 

Specifically, please look at the top of these files: [https://github.com/github/gitignore/blob/master/gradle.gitignore](gradle.gitignore) and [https://github.com/github/gitignore/blob/master/Global/macOS.gitignore](macOS.gitignore)